src/quemb/molbe/autofrag.py

```diff
@@ -92,14 +207,14 @@ class FragmentMap:
 
     fsites: list[Sequence[int]]
     fs: list[Sequence[Sequence[int]]]
-    edge: list[Sequence[Sequence[int]]]
+    edge_sites: list[Sequence[Sequence[int]]]
     center: list[Sequence[int]]
     centerf_idx: list[Sequence[int]]
-    ebe_weights: list[Sequence]
+    ebe_weight: list[Sequence]
     sites: list[Sequence]
     dnames: list[str]
-    fragment_atoms: list[Sequence[int]]
-    center_atoms: list[Sequence[int]]
+    Frag_atom: list[Sequence[int]]
+    center_atom: list[Sequence[int]]
```

Comment:
For the renamed attributes, could you also modify the docstrings? I am not sure how I feel about the name changes themselves -- @ShaunWeatherly any thoughts?

Reply:
Note that these are very similar to the `autogen` components and now just unified names.
Before the assignments read for example as:
```python3
self.ebe_weight = fragment_map.ebe_weights
```
where it is easy to miss the additional "s" on the right hand side.

Reply:
I see! Could you reflect the changes in lines 167-20 (docstrings) as well?

Reply:
the docstrings are also fixed now